src/test/java/ua/javaPro/berezhnoy/hillelHomeWork6/HillelCollectionsTest.java

```diff
@@ -0,0 +1,101 @@
+package ua.javaPro.berezhnoy.hillelHomeWork6;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class HillelCollectionsTest {
+
+    @Test
+    void shouldCountNumberMatchesTest() {
+        var hillelCollections = new HillelCollections();
+        List list = new ArrayList<String>();
+        list.add("Cat");
+        list.add("Dog");
+        list.add("Lion");
+        list.add("Fretca");
+        list.add("Chucha");
+        list.add("Chucha");
+        list.add("Bona");
+        list.add("Busia");
+        list.add("Lemon");
+        list.add("Krasunia");
+        list.add("Carrot");
+        list.add("Turtle");
+        list.add("Chucha");
+        assertEquals(hillelCollections.countOccurance(list, "Chucha"), 3);
+    }
+
+    @Test
+    void arrayToListTest() {
+        var hillelCollections = new HillelCollections();
+        Integer[] array = {1, 2, 3};
+        List<Integer> list = new ArrayList<>();
+        list.add(1);
+        list.add(2);
+        list.add(3);
+        assertEquals(hillelCollections.toList(array), list);
+    }
+
+    @Test
+    void shouldReturnUniqueNumbers() {
+        var hillelCollections = new HillelCollections();
+        List<Integer> list = new ArrayList<>();
+        list.add(1);
+        list.add(2);
+        list.add(3);
+        list.add(2);
+        list.add(4);
+        list.add(7);
+        list.add(2);
+
+        assertEquals(hillelCollections.findUnique(list).toString(), "[1, 2, 3, 4, 7]");
```

Comment:
```suggestion
        assertEquals(hillelCollections.findUnique(list), List.of(1,2,3,4,7);
```
toString краще не використовувати для цього.